recipe/meta.yaml

```diff
@@ -1,40 +1,37 @@
 {% set name = "jupytext" %}
-{% set version = "1.15.2" %}
+{% set version = "1.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c9976e24d834e991906c1de55af4b6d512d764f6372aabae45fc1ea72b589173
+  sha256: 68c7b68685e870e80e60fda8286fbd6269e9c74dc1df4316df6fe46eabc94c99
 
 build:
   number: 0
   entry_points:
     - jupytext = jupytext.cli:jupytext
-  skip: true # [py<36 or s390x]
-  script_env:
-    - BUILD_JUPYTERLAB_EXTENSION=1
+    - jupytext-config = jupytext_config.__main__:main
+  skip: true # [py<38 or s390x]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
 
 requirements:
   host:
     - python
     - pip
-    - python
-    - nodejs 18.16.0
-    - jupyter-packaging
-    - jupyterlab >=3.0,<4.0
-    - setuptools
-    - wheel
+    - hatchling >=1.5.0
+    - hatch-jupyter-builder >=0.5
+    - jupyterlab >=4
```

Comment:
I'm not sure if these two are necessary. The extension files seem to already be in the sdist...

Reply:
I don't think `jupyterlab` is necessary, but it seems `jupyter-builder` is. You get `  hatchling.plugin.exceptions.UnknownPluginError: Unknown build hook: jupyter-builder` without it.

Reply:
https://github.com/mwouts/jupytext/blob/1ef1dea7d1d63e418ecc093025510e7b941ade9c/pyproject.toml#L140